src/bno055/javaupm_bno055.i

```diff
@@ -13,6 +14,10 @@
 %ignore getLinearAcceleration(float *, float *, float *);
 %ignore getGravityVectors(float *, float *, float *);
 
+%template(floatVector) std::vector<float>;
+%template(byteVector) std::vector<uint8_t>;
+%template(intVector) std::vector<int>;
+
```

Comment:
Yes it should, we will update this and the newly added sensors also.
This sensor only has the modifications to remove the unnecessary typeVectors classes that it didn't use but were still added to the package.

\*The problem with this class is that it has a function that takes _std::vector<uint8_t>_ as input, and this is not as straight forward to solve. If we force it to take an AbstractList<Byte> and then down cast it to a byteVector it will crash if the user gives it an AbstractList<Byte> that is not actually a byteVector in disguised.
A solution for this would be to have SWIG create a new typeVector and copy the content of the AbstractList<Byte> into it before passing the typeVector down. This way the function would take an AbstractList but pass down a typeVector for further processing.


We will add the AbstractList wrapper for all **outputs** of functions, but until we resolve this, the inputs will remain as typeVectors.